src/routes/MenuItemsPage.jsx

```diff
@@ -0,0 +1,110 @@
+    const createNewOrder = (id) => {
+        return {
+            id: "PLATE00" + id,
+            orderItems: [],
+            totalPrice: 0,
+            totalCount: 0
+        }
+    }
+    let allOrders = []
+    let order = {};
+    if (!state || !state.allOrders) {
+        skipModal = false;
+        order = createNewOrder(1);
+    } else if (!state.userOrder) {
+        allOrders = state.allOrders;
+        const prevId = allOrders[allOrders.length-1].id;
+        const id = Number(prevId.slice(7))
+        order = createNewOrder( id+1);
+
+    } else {
+        order = state.userOrder;
+        allOrders = state.allOrders;
+    }
+
+    const menu = [
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+        { name: "HASS OMELETE", price: 22, portionOptions: { small: 22, medium: 25, large: 30}, description: "Bacon, avacado, monterey jack, pico de gallo, mild cheddar", image: "https://www.foodnetwork.com/content/dam/images/food/fullset/2021/07/14/0/FNK_Hash-Brown-Omelet_H1_s4x3.jpg"},
+    ]
```

Comment:
Please add a `key` prop to each of these menu item to remove the console warning. This is important because users would see the console during heuristic evaluation.

Reply:
will add